storybook/stories/staking/StakePools.stories.js

```diff
@@ -5,9 +5,11 @@ import { action } from '@storybook/addon-actions';
 
 import StakePools from '../../../source/renderer/app/components/staking/stake-pools/StakePools';
 import STAKE_POOLS from '../../../source/renderer/app/config/stakingStakePools.dummy.json';
+import DUMMY_WALLETS from '../../../source/renderer/app/config/wallets.dummy';
```

Comment:
@yakovkaravelov use `generateWallet()` storybook helper for wallets creation and remove a manually created file

Reply:
👍